plugin/build_baron.go

```diff
@@ -59,9 +62,12 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 			return errors.Errorf(`Failed validating configuration for project "%s": `+
 				"bf_suggestion_timeout_secs must be zero when bf_suggestion_url is blank", projName)
 		}
-		if proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != "" {
+		if webhookConfigured {
 			if _, err := url.Parse(proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint); err != nil {
-				return errors.Wrapf(err, `Failed to parse webhook endpoint for project "%s"`, projName)
+				grip.Error(message.WrapError(err, message.Fields{
+					"message": fmt.Sprintf(`Failed to parse webhook endpoint for project "%s"`, projName),
```

Comment:
Rather than use ` ` to wrap "", we typically use "" and ' '. That explanation may have been confusing..but basically: 
"Failed to parse webhook endpoint for project '%s'".

Also, if you're already using message.WrapError(err), you don't need to specify the error again in the message (in fact you don't want to, because now you're repeating it.

Reply:
ah yes, I don't want to wrap it. thanks!

Reply:
I don't think you pushed this fix.

Reply:
should be there now

Reply:
sorry, I was also referencing my first comment about how the message is formatted: 
> Rather than use to wrap "", we typically use "" and ' '. That explanation may have been confusing..but basically: "Failed to parse webhook endpoint for project '%s'".

Although now that you're using message.Fields, it makes more sense to also use "project_name": projName as a separate field , rather than formatting it here